go/client/chat_common.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name string) (*keybase1.UserOrTeamResult, error) {
```

Comment:
not sure if I did the error checking right, especially the `err.(Type)` syntax, but it seems to work OK.